src/algebraic_topology/simplex_category.lean

```diff
@@ -402,4 +402,149 @@ full_subcategory_inclusion _
 
 end truncated
 
+section concrete
+
+instance : concrete_category.{0} simplex_category.{u} :=
+{ forget :=
+  { obj := λ i, fin (i.len + 1),
+    map := λ i j f, f.to_preorder_hom },
+  forget_faithful := {} }
+
+end concrete
+
+section epi_mono
+
+/-- A morphism in `simplex_category` is a monomorphism precisely when it is an injective function
+-/
+theorem mono_iff_injective {n m : simplex_category} {f : n ⟶ m} :
+  mono f ↔ function.injective f.to_preorder_hom :=
+begin
+  split,
+  { introsI m x y h,
+    have H : const n x ≫ f = const n y ≫ f,
+    { dsimp, rw h },
+    change (n.const x).to_preorder_hom 0 = (n.const y).to_preorder_hom 0,
+    rw cancel_mono f at H,
+    rw H },
+    { intro hyp_inj,
+      exact concrete_category.mono_of_injective _ hyp_inj }
```

Comment:
At least re-indent:
```suggestion
  { intro hyp_inj,
    exact concrete_category.mono_of_injective _ hyp_inj }
```
And maybe also shorten to:
```suggestion
  { exact concrete_category.mono_of_injective f }
```

Reply:
Did you try this suggestion? It doesn't work!